wordpress-mobile/WordPress-Android
# Remove inline styling from reader post content

Fixes #15547

This PR adds logic to `ReaderPostRenderer` to strip all inline `style` attributes from reader post content. This is done in place using jsoup, before the html is reworked to include any Android reader specific styling. Stripping all inline styles matches iOS processing.

**Notes**
This is the very first time that Android is stripping the inline styles. In addition to testing this as a fix for buttons, I also recommend testing all blocks in the reader to ensure we haven't introduced any other rendering issues. Having a few eyes on this would be better than just mine. Thanks. cc: @reginabally @yaelirub 


**To test:**
**Prerequisites**
- Two accounts are needed. 
- Account 1: Used to create the post.
- Account 2: Is a follower of account 1 

**Test**
- Login in to WP as Account 1 (via calypso or the app) 
- Create a new post
- Add a button (or multiple buttons)
- Publish the post

- Login in to the App as Account 2
- Navigate to reader
- Tap the post to view the details
- Note that buttons are shown as expected and not as empty boxes

*** Sometimes the cache doesn't refresh enough between new post creation and the post being injected with inline styles. So instead of the above instructions, you can run the test with the following post:  https://premiumtestsite20190311.blog/2021/11/10/test-button/



## Regression Notes
1. Potential unintended areas of impact
Block rendering weirdness across the board

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
